### DISCUSSION TITLE
Rewards for the top bug reporters February 2021

### DISCUSSION THREAD

**Discussion by FrancescoBorzi** [chromiecraft/chromiecraft#146] [General]
The following users have opened several valid* bug reports during the last period and we thank them with some prizes!

- @Cosmic-Cocoa, @Cosmic-Lattee, @Istox, @Sathrenor, @claymore147, @necropola

Send me a PM on Discord to get your prize.

* we consider a bug report valid only it has been validated from a contributor and properly linked to AzerothCore (such issues get the tag Linked [AC])

**Comment by ghost** [comment]
> * we consider a bug report valid only it has been validated from a contributor and properly linked to AzerothCore (such issues get the tag Linked [AC])

MarkDown needs an escape character (backslash, (`\`)), without space, in front of any character that is read as formatting or code.

> \* We consider a bug report valid only if it has been validated by a contributor and properly linked to AzerothCore (such issues get the tag Linked [AC])

**Comment by FrancescoBorzi** [comment]
Thanks everyone for helping our project. Don't forget to Star our main repository:

https://github.com/azerothcore/azerothcore-wotlk
